lc2fen/fen.py

```diff
@@ -100,10 +100,10 @@ def is_white_square(list_pos):
     if not 0 <= list_pos <= 63:
         raise ValueError("List positions must be between 0 and 63")
 
-    if list_pos % 16 < 8:  # Odd rows
-        return list_pos % 2 == 0
-    else:  # Even rows
-        return list_pos % 2 == 1
+    if list_pos % 16 < 8:  # Rank 8, 6, 4, or 2
```

Comment:
Could you add these documentation fixes in a different PR with some rationale? Unless it's just a typo, I'd prefer not to merge this type of fixes together with new features.

Reply:
Done. I thought about creating a separate PR at the time, but I thought the change was not significant enough to warrant a new PR (since it only changes four lines of comments)